e2e/terraform/README.md

```diff
@@ -147,26 +150,55 @@ Common commands:
   - Fix these errors and re-verify with `terraform plan` before running the `terraform apply` command.
 - Run the command `terraform apply` to initiate creation of the infrastructure.
 
-## SSH to the Master Node
+## List AWS EC2 instances
+
+- From your workstation, run the following command to list the created AWS EC2 instances.
+
+```
+$ ./oebs-cloud.sh --list-aws-instances
+      Node                     Private IP Address   Public IP Address    
+nodes.openebs.k8s.local          172.20.36.126        54.90.239.23         
+nodes.openebs.k8s.local          172.20.37.115        34.224.169.116       
+masters.openebs.k8s.local        172.20.53.140        34.202.205.27        
+
+```
+
+## SSH to the Kubernetes Node
 
 - From your workstation, run the following command to connect to the EC2 instance running the Kubernetes Master.
 
+**For Ubuntu**
+
 ```
 $ ./oebs-cloud.sh --ssh-aws-ec2
+Welcome to Ubuntu 16.04 LTS (GNU/Linux 4.4.0-93-generic x86_64)
+ubuntu@ip-172-20-53-140 ~ $
 ```
 
-- You should now be running inside the EC2 instance.
+**For CoreOS**
+
+```
+$ ./oebs-cloud.sh --ssh-aws-ec2
+Container Linux by CoreOS stable (1465.6.0)
+core@ip-172-20-53-140 ~ $
+```
+
+- Running `--ssh-aws-ec2` without any arguments, by default connects you to the Kubernetes Master.
+    - `--ssh-aws-ec2` can also be run as `--ssh-aws-ec2=ipaddress`, where `ipaddress` is the Public IPAddress of the AWS EC2 instance.
+- You should now be running inside the AWS EC2 instance.
 
 ## Deploy OpenEBS on AWS
 
 Deploying OpenEBS requires Kubernetes to be already running on the EC2 instances. Verify if a Kubernetes cluster has been created.
```

Comment:
replace to be already = Kubernetes must be running on the EC2 instances while deploying OpenEBS. 
replace has been = Verify if a Kubernetes cluster is created.